processing/data_collection/gazette/spiders/sc_florianopolis.py

```diff
@@ -21,16 +21,23 @@ def start_requests(self):
             year, month = str(target.year), str(target.month)
             data = dict(ano=year, mes=month, passo="1", enviar="")
             yield FormRequest(url=self.URL, formdata=data, callback=self.parse)
-            target = target + relativedelta(months=1)
+            if target <= self.start_date:
```

Comment:
I think it would be safer to check for **self.start_date** before trying to use it, as this attribute could be None and it would break the spider.

Reply:
Nice catch!